tesseract/tesseract_planning/tesseract_motion_planners/test/simple_planner_fixed_size_assign_position.cpp

```diff
@@ -83,24 +83,107 @@ class TesseractPlanningSimplePlannerFixedSizeAssignPositionUnit : public ::testi
   }
 };
 
-TEST_F(TesseractPlanningSimplePlannerFixedSizeAssignPositionUnit, XXXXX)  // NOLINT
+TEST_F(TesseractPlanningSimplePlannerFixedSizeAssignPositionUnit, Eigen_AssignJointPosition)  // NOLINT
 {
-  //  PlannerRequest request;
-  //  request.tesseract = tesseract_ptr_;
-  //  request.env_state = tesseract_ptr_->getEnvironmentConst()->getCurrentState();
-  //  JointWaypoint wp1 = Eigen::VectorXd::Zero(7);
-  //  JointWaypoint wp2 = Eigen::VectorXd::Ones(7);
-  //  PlanInstruction instr(wp1, PlanInstructionType::FREESPACE, "DEFAULT", manip_info_);
-
-  //  auto composite = fixedSizeJointInterpolation(wp1, wp2, instr, request, ManipulatorInfo(), 10);
-  //  EXPECT_EQ(composite.size(), 10);
-  //  for (const auto& c : composite)
-  //  {
-  //    EXPECT_TRUE(isMoveInstruction(c));
-  //    EXPECT_TRUE(isStateWaypoint(c.cast_const<MoveInstruction>()->getWaypoint()));
-  //  }
-  //  const auto* mi = composite.back().cast_const<MoveInstruction>();
-  //  EXPECT_TRUE(wp2.isApprox(mi->getWaypoint().cast_const<StateWaypoint>()->position, 1e-5));
+  PlannerRequest request;
+  request.tesseract = tesseract_ptr_;
+  request.env_state = tesseract_ptr_->getEnvironmentConst()->getCurrentState();
+  Eigen::VectorXd vec1 = Eigen::VectorXd::Zero(7);
+  JointWaypoint wp1 = vec1;
+  PlanInstruction instr(wp1, PlanInstructionType::FREESPACE, "DEFAULT", manip_info_);
+  auto composite = fixedSizeAssignJointPosition(vec1, instr, request, ManipulatorInfo(), 10);
+  EXPECT_EQ(composite.size(), 10);
+  for (const auto& c : composite)
+  {
+    EXPECT_TRUE(isMoveInstruction(c));
```

Comment:
I think this should be assert true. If this is false the next line will segfault wont it?